charmander/app.py

```diff
@@ -37,7 +37,7 @@ def create_app(debug=False, testing=False, model_only=False):
     )
 
     graph.use(
```

Comment:
The `microcosm.graph` is our `microcosm` for the whole app itself, and as such has references to every significant component in the application. To be able to instantiate the components mentioned here, the `graph` needs factories for those components to be registered with it. The `binding` decorator used through out takes care of assigning factories to the `graph` registry, but we still need to import the files where the `binding` decorator is used so the registration takes place.